src/init.cpp

```diff
@@ -1012,20 +1012,20 @@ bool AppInitParameterInteraction(const ArgsManager& args)
 
     // Sanity check argument for min fee for including tx in block
     // TODO: Harmonize which arguments need sanity checking and where that happens
```

Comment:
Is this TODO still accurate?  Not what arguments need Harmonizing.

Reply:
I agree the TODO isn't very clear and possibly no longer applicable.  Maybe it could be rounded up with other unclear comments and TODOs in a separate PR.  A PR to just remove this one TODO line seems silly.